titus-server-master/src/main/java/com/netflix/titus/master/appscale/service/DefaultAppScaleManager.java

```diff
@@ -167,6 +178,12 @@ public Completable enterActiveMode() {
         checkForScalingPolicyActions().toCompletable().await(appScaleManagerConfiguration.getStoreInitTimeoutSeconds(),
                 TimeUnit.SECONDS);
 
+        reconcileAllPendingRequests = Observable.interval(appScaleManagerConfiguration.getReconcileAllPendingAndDeletingRequestsIntervalMins(), TimeUnit.MINUTES)
```

Comment:
I am going to defer converting all scheduled actions to using ObervableExt.schedule to the non-hotfix release later.